src/Node/Expression/CallExpression.php

```diff
@@ -112,7 +112,7 @@ protected function compileArguments(Compiler $compiler, $isArray = false): void
         $compiler->raw($isArray ? ']' : ')');
     }
 
-    protected function getArguments($callable = null, $arguments)
+    protected function getArguments($callable = null, $arguments = [])
```

Comment:
I'd rather remove `= null` here (I guess that there was a typehint at some point justifying the `= null` to make the typehint nullable, and it was removed later)